pdc-client.spec

```diff
@@ -231,6 +231,30 @@ EOF
 
 
 %changelog
+* Wed Nov 22 2017 Chuang Cao <chcao@redhat.com> 1.8.0-1
+- Add PDCClient tests and fix the discovered bugs (lholecek@redhat.com)
+- Add comments for rpm requirements (chcao@redhat.com)
+- Get the endpoint as attr which includes "-" (chcao@redhat.com)
+- Fix wrapping BeanBag methods and operators (lholecek@redhat.com)
+- Fix bug of Multipaged (chcao@redhat.com)
+- Add MultiPageBeanBag class to support get multi pages (chcao@redhat.com)
+- Add the close function when load plugins (chcao@redhat.com)
+- Correct the flake8 issues (chcao@redhat.com)
+- Change the docstrings and fix issues (chcao@redhat.com)
+- Remove duplicate code (lholecek@redhat.com)
+- Update documentation (lholecek@redhat.com)
+- Add documentation link to README file (lholecek@redhat.com)
+- Add discription of page_size=-1 in help doc (chcao@redhat.com)
+- Fix printing errors and exit code for pdc_client (lholecek@redhat.com)
+- Add smoke-test for all executables (lholecek@redhat.com)
+- Fix running tests when building rpm (lholecek@redhat.com)
+- Override plugin paths with PDC_CLIENT_PLUGIN_PATH (lholecek@redhat.com)
+- Improve installing plugins (lholecek@redhat.com)
+- Revert removing comments from downstream (lholecek@redhat.com)
+- Automatic commit of package [pdc-client] minor release [1.7.0-2].
```

Comment:
Remove this line.

Reply:
Remove it. The commit is also in release branch, not sure how it got in master before.

Tito adds all new commits. It only tries to help you write better changelog. Some chagelog entries are only about refactoring or fixing issues that did not appear in previous versions (happened during development of new version) - these are not relevant to the last version.

Reply:
Yes, I know that.

Just thinking to keep the log here for discussion if there is a need to keep, I also prefer to remove this.